packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -466,10 +475,33 @@ export const Webchat = forwardRef((props, ref) => {
     undefined
   )
 
+  const [hover, setHover] = useState(false)
+
+  const TriggerButtonHoverOpacity = hover
+    ? getThemeProperty(
+        'triggerButton.hoverOpacity',
+        getThemeProperty('triggerButton.style.opacity', 1)
+      )
+    : getThemeProperty('triggerButton.style.opacity', 1)
+  const TriggerButtonHoverColor = hover
+    ? getThemeProperty(
+        'triggerButton.hoverColor',
+        getThemeProperty('triggerButton.style.background', COLORS.SOLID_WHITE)
+      )
+    : getThemeProperty('triggerButton.style.background', COLORS.SOLID_WHITE)
```

Comment:
have you tried to use the `CustomTriggerButton` here with on hover styles instead of adding properties to the "default" trigger button?